cypress/e2e/data-import/settings/verify-material-type-inactive-in-open-order.cy.js

```diff
@@ -0,0 +1,158 @@
+import {
+  FOLIO_RECORD_TYPE,
+  ORDER_STATUSES,
+  VENDOR_NAMES,
+  ACQUISITION_METHOD_NAMES,
+  ORDER_FORMAT_NAMES_IN_PROFILE,
+} from '../../../support/constants';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import FieldMappingProfiles from '../../../support/fragments/data_import/mapping_profiles/fieldMappingProfiles';
+import NewFieldMappingProfile from '../../../support/fragments/data_import/mapping_profiles/newFieldMappingProfile';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import Users from '../../../support/fragments/users/users';
+import { Permissions } from '../../../support/dictionary';
+import SettingsMappingProfiles from '../../../support/fragments/settings/dataImport/settingsMappingProfiles';
+import FieldMappingProfileView from '../../../support/fragments/data_import/mapping_profiles/fieldMappingProfileView';
+import FieldMappingProfileEdit from '../../../support/fragments/data_import/mapping_profiles/fieldMappingProfileEdit';
+
+describe('data-import', () => {
+  describe('Settings', () => {
+    const testData = {};
+    const mappingProfileC380560 = {
+      name: `C380560testMappingProfile.${getRandomPostfix()}`,
+      typeValue: FOLIO_RECORD_TYPE.ORDER,
+      orderStatus: ORDER_STATUSES.OPEN,
+      incomingRecordType: 'MARC_BIBLIOGRAPHIC',
+      recordType: 'ORDER',
+      description: '',
+    };
+    const mappingProfileC380561 = {
+      name: `C380561testMappingProfile.${getRandomPostfix()}`,
+      typeValue: FOLIO_RECORD_TYPE.ORDER,
+      orderStatus: ORDER_STATUSES.OPEN,
+      vendor: VENDOR_NAMES.GOBI,
+      title: '245$a',
+      acquisitionMethod: ACQUISITION_METHOD_NAMES.PURCHASE_AT_VENDOR_SYSTEM,
+      orderFormat: ORDER_FORMAT_NAMES_IN_PROFILE.PE_MIX,
+      receivingWorkflow: 'Synchronized',
+      currency: 'USD',
+      incomingRecordType: 'MARC_BIBLIOGRAPHIC',
+      recordType: 'ORDER',
+      description: '',
+    };
+    const fields = {
+      purchaseOrderStatus: { accordion: 'Order information', fieldName: 'Purchase order status*' },
+      materialTypePhysical: { accordion: 'Physical resource details', fieldName: 'Material type' },
+      materialTypeElectronic: { accordion: 'E-resources details', fieldName: 'Material type' },
+    };
+
+    before('Create test data', () => {
+      cy.createTempUser([
+        Permissions.settingsDataImportEnabled.gui,
+        Permissions.uiOrganizationsView.gui,
+      ]).then((userProperties) => {
+        testData.user = userProperties;
+        cy.login(testData.user.username, testData.user.password, {
+          path: SettingsMenu.mappingProfilePath,
+          waiter: FieldMappingProfiles.waitLoading,
+        });
+      });
+    });
+
+    after('Delete test data', () => {
+      cy.getAdminToken();
+      Users.deleteViaApi(testData.user.userId);
+      FieldMappingProfiles.getFieldMappingProfileInDataImport({
```

Comment:
For deleting mapping profiles is the function as FieldMappingProfileView.deleteViaApi(profileName); Please use it